src/shared/schema/avl.schema.ts

```diff
@@ -222,7 +222,7 @@ export const siriSchemaTransformed = (errors?: AvlValidationError[]) =>
                     vehicleActivity.Extensions?.VehicleJourney?.Operational?.TicketMachine?.JourneyCode ?? null,
                 vehicle_unique_id: vehicleActivity.Extensions?.VehicleJourney?.VehicleUniqueId ?? null,
                 has_onward_calls: !!vehicleActivity.MonitoredVehicleJourney.OnwardCalls,
```

Comment:
Can we remove the `has_onward_calls` column now? I think it was used in the code to know when to add items to the onwards calls table but I can't see any use for it in the code now.

Reply:
Removed it and added in a migration to drop the column and the avl_onward_call table